src/ViewRendererBundle/Bridge/Twig/Loader/TwigStringLoader.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace ChameleonSystem\ViewRendererBundle\Bridge\Twig\Loader;
+
+/**
+ * Loads a template from a string.
+ *
+ * This loader should NEVER be used. It only exists for Twig internal purposes.
```

Comment:
Why is this class in our bundle?

Reply:
It's because it was removed from Twig in version 2.0 and we still need it (could have re-implemented it, but didn't want to hide the fact that it is basically the same class as was used before and written by others).